build/README.md

```diff
@@ -0,0 +1,3 @@
+# Build tools project
+
+The build module is the first module built as part of the overall project reactor build. The purpose is to enable utilities that will be relevant in the build of other projects. It currently serves as the repository for the static code analysis tool PMD's ruleset definition that is shared across all modules in the build. See https://maven.apache.org/plugins/maven-pmd-plugin/examples/multi-module-config.html for details.
```

Comment:
This is probably a question for the team, but do we want to limit line length in our markdown?

I know the actual rendered pages are line wrapped regardless, but shorter lines make it easier to see line changes as markdown files are updated over time. If we did line length it would be strictly for easier to read diffs.